fitbenchmarking/utils/options.py

```diff
@@ -104,6 +105,7 @@ class Options:
     DEFAULT_MINIMZERS = \
         {'bumps': ['amoeba', 'lm-bumps', 'newton', 'scipy-leastsq'],
          'dfo': ['dfogn', 'dfols'],
+         'gofit': ['alternating', 'multistart'],
```

Comment:
Should alternating be a default if it's only suitable for CrystalField? Also is regularisation slow or just not good as a default for some other reason?

Reply:
Agree alternating should not be default, removed. I feel that having a local optimiser (regularisation) as a default for a global optimization package is a bad default.